server/routes/events.test.js

```diff
@@ -109,7 +101,6 @@ describe('GET /api/v1/events/:id', () => {
     })
     return request(server)
       .get('/api/v1/events/2')
-      .set(testAuthAdminHeader)
```

Comment:
This test looks like it is testing an admin scenario. Doesn't it need the admin token?